### PR TITLE
Implement dark mode on get started and language

### PR DESCRIPTION
**Pull request by oyenuga17**
@sabine I've closed #1897 and replaced with this. I messed that one up when trying out rebase lol, but I think I'm getting a hang of it now. Please review this first before the others.

### CODE REVIEW

File: src/ocamlorg_frontend/components/footer.eml [hunk 1]
```diff
@@ -43,14 +43,14 @@ let socials = [
 
 let primary_footer () =
   let icon_link ~href ~name ~icon =
-    <a href="<%s href %>" class="text-content dark:text-dark-title hover:text-primary dark:hover:text-dark-primary">
+    <a href="<%s href %>" class="text-content dark:text-dark-title hover:text-tertiary_25 dark:hover:text-dark-primary">
```
Comment (sabine) [inline src/ocamlorg_frontend/components/footer.eml]:
```suggestion
    <a href="<%s href %>" class="text-content dark:text-dark-title hover:text-primary dark:hover:text-dark-primary">
```

File: src/ocamlorg_frontend/components/footer.eml [hunk 2]
```diff
@@ -43,14 +43,14 @@ let socials = [
...
       <span class="sr-only"><%s name %></span>
       <%s! icon "h-6 w-6" %>
     </a>
   in
   let footer_link ~href ~name =
     <li>
-      <a href="<%s href %>" class="text-base leading-6 text-content dark:text-dark-title hover:text-primary dark:hover:text-dark-primary"><%s name %></a>
+      <a href="<%s href %>" class="text-base leading-6 text-content dark:text-dark-title hover:text-tertiary_25 dark:hover:text-dark-primary"><%s name %></a>
```
Comment (sabine) [inline src/ocamlorg_frontend/components/footer.eml]:
```suggestion
      <a href="<%s href %>" class="text-base leading-6 text-content dark:text-dark-title hover:text-primary dark:hover:text-dark-primary"><%s name %></a>
```

File: src/ocamlorg_frontend/components/forms.eml [hunk 3]
```diff
@@ -10,7 +10,7 @@ let search_input ?button_attrs ?input_attrs ?dropdown_html ~label ~name _class =
         type="search"
         name="<%s name %>"
         placeholder="<%s label %>"
-        class="bg-white dark:bg-dark-card h-full w-full font-medium focus:border-primary dark:focus:border-dark-primary focus:ring-0 text-gray-800 dark:text-dark-title border-primary dark:border-dark-primary rounded-md rounded-r-none px-3 py-1 placeholder-text-content dark:placeholder-dark-title appearance-none focus:outline-none"
+        class="bg-white dark:bg-dark-card h-full w-full font-medium focus:border-primary dark:focus:border-dark-primary focus:ring-0 text-gray-800 border-primary dark:border-dark-primary rounded-md rounded-r-none px-3 py-1 placeholder-text-content dark:placeholder-dark-title appearance-none focus:outline-none"
```
Comment (sabine) [inline src/ocamlorg_frontend/components/forms.eml]:
```suggestion
        class="bg-white text-title dark:text-dark-title dark:bg-dark-card h-full w-full font-medium focus:border-primary dark:focus:border-dark-primary focus:ring-0 text-gray-800 border-primary dark:border-dark-primary rounded-md rounded-r-none px-3 py-1 placeholder-text-content dark:placeholder-dark-title appearance-none focus:outline-none"
```

File: src/ocamlorg_frontend/components/header.eml [hunk 4]
```diff
@@ -121,13 +121,13 @@ in
       </li>
       <li>
         <div class="space-x-6 text-2xl flex items-center">
-          <a aria-label="OCaml's Discord" href="https://discord.gg/cCYQbqN" class="opacity-60 hover:opacity-100 text-content dark:text-dark-title hover:text-primary dark:hover:text-dark-primary">
+          <a aria-label="OCaml's Discord" href="https://discord.gg/cCYQbqN" class="opacity-60 hover:opacity-100 text-content dark:text-dark-title hover:text-tertiary_25 dark:hover:text-dark-primary">
```
Comment (sabine) [inline src/ocamlorg_frontend/components/header.eml]:
```suggestion
          <a aria-label="OCaml's Discord" href="https://discord.gg/cCYQbqN" class="opacity-60 hover:opacity-100 text-content dark:text-dark-title hover:text-primary dark:hover:text-dark-primary">
```

File: src/ocamlorg_frontend/components/header.eml [hunk 5]
```diff
@@ -121,13 +121,13 @@ in
...
             <%s! Icons.discord "w-6 h-6" %>
           </a>
-          <a aria-label="The OCaml Compiler on GitHub" href="https://github.com/ocaml/ocaml" class="opacity-60 hover:opacity-100 text-content dark:text-dark-title hover:text-primary dark:hover:text-dark-primary">
+          <a aria-label="The OCaml Compiler on GitHub" href="https://github.com/ocaml/ocaml" class="opacity-60 hover:opacity-100 text-content dark:text-dark-title hover:text-tertiary_25 dark:hover:text-dark-primary">
```
Comment (sabine) [inline src/ocamlorg_frontend/components/header.eml]:
```suggestion
          <a aria-label="The OCaml Compiler on GitHub" href="https://github.com/ocaml/ocaml" class="opacity-60 hover:opacity-100 text-content dark:text-dark-title hover:text-primary dark:hover:text-dark-primary">
```

File: src/ocamlorg_frontend/components/header.eml [hunk 6]
```diff
@@ -121,13 +121,13 @@ in
...
             <%s! Icons.github "w-6 h-6" %>
           </a>
-          <a aria-label="The OCaml Language Twitter Account" href="https://twitter.com/ocamllang" class="opacity-60 hover:opacity-100 text-content dark:text-dark-title hover:text-primary dark:hover:text-dark-primary">
+          <a aria-label="The OCaml Language Twitter Account" href="https://twitter.com/ocamllang" class="opacity-60 hover:opacity-100 text-content dark:text-dark-title hover:text-tertiary_25 dark:hover:text-dark-primary">
```
Comment (sabine) [inline src/ocamlorg_frontend/components/header.eml]:
```suggestion
          <a aria-label="The OCaml Language Twitter Account" href="https://twitter.com/ocamllang" class="opacity-60 hover:opacity-100 text-content dark:text-dark-title hover:text-primary dark:hover:text-dark-primary">
```

File: src/ocamlorg_frontend/pages/learn.eml [hunk 7]
```diff
@@ -8,7 +8,7 @@ Learn_layout.single_column_layout
 ~canonical:Url.learn
 ~active_top_nav_item:Header.Learn
 ~current:Overview @@
-  <div class="w-full border-b bg-[#DDEAFC] dark:bg-[#101B3F] md:bg-transparent dark:border-none md:bg-[length:100%_100%] md:bg-[url('/img/learn/hero-background.svg')] dark:md:bg-[url('/img/learn/hero-background-dark.svg')]">
+  <div class="w-full border-b bg-[#DDEAFC] dark:bg-[#101B3F] dark:border-none md:bg-[length:100%_100%] md:bg-[url('/img/learn/hero-background.svg')] dark:md:bg-[url('/img/learn/hero-background-dark.svg')]">
```
Comment (sabine) [inline src/ocamlorg_frontend/pages/learn.eml]:
```suggestion
  <div class="w-full border-b bg-[#DDEAFC] dark:bg-[#101B3F] md:bg-transparent dark:border-none md:bg-[length:100%_100%] md:bg-[url('/img/learn/hero-background.svg')] dark:md:bg-[url('/img/learn/hero-background-dark.svg')]">
```

**Review by sabine — APPROVED** [review]
Thanks @oyenuga17, looks good to me! :sparkles: :heart:

If there's inaccuracies we overlooked, we can fix them later. I'll commit and merge these minor corrections.

**Comment by oyenuga17** [comment]
Finally 😅

**Pull request merged**